index.html

```diff
@@ -1,13 +1,52 @@
+                            d="m32.542 18.443v4.64h-3.2968v-13.676h5.4571c1.0457 0 1.9693 0.19412 2.7708 0.58235 0.8078 0.38194 1.4309 0.92984 1.8692 1.6437 0.4446 0.7075 0.6669 1.5122 0.6669 2.4139 0 1.3337-0.4791 2.4014-1.4371 3.2029-0.9518 0.7952-2.2605 1.1928-3.9261 1.1928h-2.104zm0-2.5454h2.1603c0.6387 0 1.124-0.1597 1.4559-0.479 0.3381-0.3194 0.5072-0.7702 0.5072-1.3525 0-0.6387-0.1722-1.1491-0.5166-1.531-0.3444-0.382-0.814-0.5761-1.4089-0.5824h-2.1979v3.9449z" />
+                        <path
+                            d="m46.28 18.256h-1.7846v4.8278h-3.2968v-13.676h5.382c1.6218 0 2.8898 0.36006 3.804 1.0802 0.9142 0.7201 1.3713 1.7376 1.3713 3.0526 0 0.9518-0.1941 1.7407-0.5823 2.3669-0.382 0.6262-0.9831 1.1334-1.8034 1.5216l2.8553 5.5135v0.1409h-3.5316l-2.4139-4.8278zm-1.7846-2.5454h2.0852c0.6261 0 1.0958-0.1628 1.4089-0.4884 0.3193-0.3319 0.479-0.7922 0.479-1.3808s-0.1597-1.0488-0.479-1.3807c-0.3194-0.3381-0.789-0.5072-1.4089-0.5072h-2.0852v3.7571z" />
+                        <path d="m57.067 23.083h-3.2874v-13.676h3.2874v13.676z" />
+                        <path
+                            d="m64.266 9.4077l3.0526 9.5147 3.0432-9.5147h4.3394v13.676h-3.3062v-3.1935l0.3194-6.5373-3.3062 9.7308h-2.1791l-3.3156-9.7402 0.3193 6.5467v3.1935h-3.2968v-13.676h4.33z" />
+                        <path d="m87.062 17.654h-9.9092v-2.4796h9.9092v2.4796z" />
+                        <path
+                            d="m100 9.4077v8.9512c0 1.0144-0.216 1.8942-0.6481 2.6393-0.4321 0.7389-1.052 1.3025-1.8597 1.6907-0.8078 0.3882-1.7627 0.5823-2.8648 0.5823-1.6656 0-2.9774-0.432-3.9355-1.2961-0.958-0.8642-1.4465-2.0476-1.4652-3.5505v-9.0169h3.3156v9.0827c0.0375 1.4965 0.7326 2.2448 2.0851 2.2448 0.6826 0 1.1991-0.1878 1.5498-0.5635s0.526-0.9863 0.526-1.8316v-8.9324h3.2968z" />
+                        <path class="header__logo-accent"
+                            d="m10.259 32.701c7.6944-8.9769 10.259-18.274 10.259-22.122-0.3206-7.0533-5.7708-10.58-10.259-10.58v5.7708c2.4789 0 4.4884 2.0095 4.4884 4.4885 0 2.4788-2.0095 4.4884-4.4884 4.4884v17.954z"
+                            clip-rule="evenodd" fill-rule="evenodd" />
+                        <path
+                            d="m10.259 32.701c-7.6944-8.9769-10.259-18.274-10.259-22.122 0.3206-7.0533 5.7708-10.58 10.259-10.58l1e-4 5.7708c-2.4789 0-4.4885 2.0095-4.4885 4.4885 0 2.4788 2.0095 4.4884 4.4885 4.4884l-1e-4 17.954z"
+                            clip-rule="evenodd" fill-rule="evenodd" />
+                        <path
+                            d="m10.259 20.518c5.666 0 10.259-4.5932 10.259-10.259 0-5.6661-4.5932-10.259-10.259-10.259-5.6661 0-10.259 4.5932-10.259 10.259 0 5.666 4.5932 10.259 10.259 10.259zm0-5.7708c2.4788 0 4.4884-2.0096 4.4884-4.4884 0-2.4789-2.0096-4.4885-4.4884-4.4885-2.4789 0-4.4885 2.0095-4.4885 4.4885 0 2.4788 2.0095 4.4884 4.4885 4.4884z"
+                            clip-rule="evenodd" fill-rule="evenodd" />
+                    </svg>
```

Comment:
I don't like using inline SVG's but in this case I couldn't figure out how to change SVG colors on hover/scroll without `:hover` and modifier classes attached to the `svg` and its paths. I think I could use an `img` tag then replace the image background on hover or the image source on mouseover but I wasn't sure that I could do that _and_ implement a smooth transition between the colors. 

Looks like I could do something like [this](https://stackoverflow.com/a/13034571/15154619)? Think it's worth going back and refactoring or can that be a later thing?

Reply:
[here](https://stackoverflow.com/questions/23583245/add-transition-while-changing-img-src-with-javascript) has one answer where they have both images and use opacity to have one show, and switch on the hover. (could probably also do similar things with other attributes like display and visibility which you can set a transition on too)
Changing out the source for an image might be easier done with just css via background-image styles rather than an image tag, but img would be more appropriate given what it is. It looks like background-image is not animateable tag as far as transitions go, so getting it smooth would be difficult with that.